bfasst/design.py

```diff
@@ -23,15 +23,22 @@ class Design:
     "class holding paths and other metadata for a given design" ""
 
     def __init__(self, dir_path, experiment_dir):
-        if not dir_path.is_dir() and (paths.DESIGNS_PATH / dir_path).is_dir():
+        if not dir_path.is_dir() and dir_path.is_absolute():
+            dir_path = dir_path
+        elif not dir_path.is_dir() and (paths.DESIGNS_PATH / dir_path).is_dir():
```

Comment:
Perhaps you want to add `not dir_path.is_absolute()` to this if statement?

Reply:
reverted